website/server/controllers/api-v3/tasks.js

```diff
@@ -530,6 +530,14 @@ api.scoreTask = {
 
     if (!task) throw new NotFound(res.t('taskNotFound'));
 
+    if (task.completed && (task.type === 'daily' || task.type === 'todo') && direction === 'up') {
+      throw new NotAuthorized('Your session is outdated. Please refresh the page.');
+    }
+
+    if (!task.completed && (task.type === 'daily' || task.type === 'todo') && direction === 'down') {
+      throw new NotAuthorized('Your session is outdated. Please refresh the page.');
```

Comment:
See above where the error message is `res.t('taskNotFound')`? In one of the locale files in the `en` directory, there is a string defined with the key 'taskNotFound'.

What you need to do:
- add a new string to a file in the `en` locales directory (ideally, whichever file has the rest of the error strings) called something like 'sessionOutdated'
- set the value of the string to your error string ('Your session is outdated. Please refresh the page.')
- use `res.t('sessionOutdated')` instead of your string in the tasks code